lib/SILOptimizer/Transforms/Outliner.cpp

```diff
@@ -0,0 +1,1306 @@
+    if (auto Func = dyn_cast<FuncDecl>(Member)) {
+      Requirement = Func;
+      break;
+    }
+  }
+  assert(Requirement);
+  if (!Requirement)
+    return SILDeclRef();
+
+  auto Witness = Conformance->getWitnessDecl(Requirement, nullptr);
+  return SILDeclRef(Witness);
+}
+
+/// Get the _unconditionallyBridgeFromObjectiveC witness for the type.
+SILDeclRef getBridgeFromObjectiveC(CanType NativeType,
+                                   ModuleDecl *SwiftModule) {
+  auto &Ctx = SwiftModule->getASTContext();
+  auto Proto = Ctx.getProtocol(KnownProtocolKind::ObjectiveCBridgeable);
+  if (!Proto)
+    return SILDeclRef();
+  auto ConformanceRef =
+      SwiftModule->lookupConformance(NativeType, Proto, nullptr);
+  if (!ConformanceRef)
+    return SILDeclRef();
+  auto Conformance = ConformanceRef->getConcrete();
+  FuncDecl *Requirement = nullptr;
+  // _unconditionallyBridgeFromObjectiveC
+  DeclName Name(Ctx, Ctx.getIdentifier("_unconditionallyBridgeFromObjectiveC"),
+                llvm::makeArrayRef(Identifier()));
+  for (auto Member : Proto->lookupDirect(Name, true)) {
+    if (auto Func = dyn_cast<FuncDecl>(Member)) {
+      Requirement = Func;
+      break;
+    }
+  }
+  assert(Requirement);
+  if (!Requirement)
+    return SILDeclRef();
+
+  auto Witness = Conformance->getWitnessDecl(Requirement, nullptr);
+  return SILDeclRef(Witness);
+}
+
+/// Pattern for a bridged property call.
+///
+///  bb7:
+///    %30 = unchecked_take_enum_data_addr %19 : $*Optional<UITextField>, #Optional.some!enumelt.1
+///    %31 = load %30 : $*UITextField
+///    strong_retain %31 : $UITextField
+///    %33 = class_method [volatile] %31 : $UITextField, #UITextField.text!getter.1.foreign : (UITextField) -> () -> String?, $@convention(objc_method) (UITextField) -> @autoreleased Optional<NSString>
+///    %34 = apply %33(%31) : $@convention(objc_method) (UITextField) -> @autoreleased Optional<NSString>
+///    switch_enum %34 : $Optional<NSString>, case #Optional.some!enumelt.1: bb8, case #Optional.none!enumelt: bb9
+///
+///  bb8(%36 : $NSString):
+///    // function_ref static String._unconditionallyBridgeFromObjectiveC(_:)
+///    %37 = function_ref @_T0SS10FoundationE36_unconditionallyBridgeFromObjectiveCSSSo8NSStringCSgFZ : $@convention(method) (@owned Optional<NSString>, @thin String.Type) -> @owned String
+///    %38 = enum $Optional<NSString>, #Optional.some!enumelt.1, %36 : $NSString
+///    %39 = metatype $@thin String.Type
+///    %40 = apply %37(%38, %39) : $@convention(method) (@owned Optional<NSString>, @thin String.Type) -> @owned String
+///    %41 = enum $Optional<String>, #Optional.some!enumelt.1, %40 : $String
+///    br bb10(%41 : $Optional<String>)
+///
+///  bb9:
+///    %43 = enum $Optional<String>, #Optional.none!enumelt
+///    br bb10(%43 : $Optional<String>)
+///
+///  bb10(%45 : $Optional<String>):
+class BridgedProperty : public OutlinePattern {
+  std::string outlinedName;
```

Comment:
Question, why capitalize this field differently?

Reply:
Too much time in IRGen ...